modules/social_features/social_embed/social_embed.install

```diff
@@ -41,14 +39,55 @@ function social_embed_install(bool $is_syncing) {
     social_embed_update_11003();
     social_embed_update_11004();
   }
+
+  // Add the embed button to the toolbar.
+  $config_factory = \Drupal::configFactory();
+  $text_formats = [
+    'editor.editor.full_html',
+    'editor.editor.basic_html',
+  ];
+
+  // Make sure our url_embed is installed, so we can add it to the toolbar.
+  \Drupal::service('module_installer')->install(['url_embed']);
+
+  foreach ($text_formats as $text_format) {
+    $configuration = $config_factory->getEditable($text_format);
+    $toolbar_items = $configuration->getOriginal('settings.toolbar.items');
+
+    // Only add the embed button when it is not on the toolbar yet.
+    if (!empty($toolbar_items) && !in_array('urlembed', $toolbar_items, TRUE)) {
+      $toolbar_items[] = 'urlembed';
+      $configuration->set('settings.toolbar.items', $toolbar_items);
+      $configuration->save();
+    }
+  }
+}
+
+/**
+ * Implements hook_uninstall().
+ *
+ * Remove embed button from the toolbar.
+ */
+function social_embed_uninstall() :void {
+  $config_factory = \Drupal::configFactory();
+  $configuration = $config_factory->getEditable('editor.editor.full_html');
```

Comment:
During the hook-install the embeb are be added to basic-html and full-html, but here only full-html is uninstalling.